google/resumable_media/_helpers.py

```diff
@@ -384,10 +384,13 @@ def _get_connection_error_classes():
     unavailable."""
 
     import requests.exceptions
+    import urllib3.exceptions
 
     return (
         requests.exceptions.ConnectionError,
         requests.exceptions.ChunkedEncodingError,
+        urllib3.exceptions.ProtocolError,
+        ConnectionError,  # Python 3.x only, superclass of ConnectionResetError.
     )
```

Comment:
Given that we are adding a stdlib type here, it seems like perhaps having this method raise `ImportError` is no longer "correct":  instead, perhaps the imports themselves should be conditional.